app/models/topic_converter.rb

```diff
@@ -63,7 +63,9 @@ def convert_to_private_message
   private
 
   def posters
-    @posters ||= @topic.posts.distinct.pluck(:user_id)
+    @posters ||= @topic.posts
+      .where.not(post_type: [Post.types[:small_action], Post.types[:whisper]])
```

Comment:
Should this be an `allowlist` instead of a `denylist`?

```suggestion
      .where.(post_type: Post.types[:regular])
```

Reply:
Not sure about `moderator_action` though 🤔

Reply:
I do not have a strong opinion about either allowlist or denylist. If we will ever add a new post type, many things will break anyway including this even if it is a small action or a whisper. From what I have seen, we use denylist in most cases `return if whisper` or `if !whisper ...`.

I was thinking about `moderator_action` too, but as far as I know it is always visible so why not invite the mod too?